src/main/frontend/webpack.config.js

```diff
@@ -5,7 +5,7 @@ const path = require("path");
 
 module.exports = {
   entry: {
-    path: "./client/main.js",
+    path: "../client/main.js",
```

Comment:
are we sure you want this? the path seems to be `frontend/client/main.js` that we're trying to get to, and you're already in the `frontend` folder

Reply:
I don't think we are sure we want this. Reverted back in another commit